frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/radiogram-card/radiogram-card-content-fallback/radiogram-card-content-fallback.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+    selector: 'app-radiogram-card-content-fallback',
+    templateUrl: './radiogram-card-content-fallback.component.html',
+    styleUrls: ['./radiogram-card-content-fallback.component.scss'],
+})
+export class RadiogramCardContentFallbackComponent {
+    @Input() radiogramType!: string;
```

Comment:
Why does it need this?

Reply:
The fallback displays the type in the UI so we can see which radiograms we forgot to add components for